include/wx/gtk/mimetype.h

```diff
@@ -23,7 +23,11 @@
 class WXDLLIMPEXP_CORE wxGTKMimeTypesManagerImpl : public wxMimeTypesManagerImpl
 {
 protected:
+#if defined(__UNIX__)
     wxString GetIconFromMimeType(const wxString& mime) wxOVERRIDE;
+#else
+    wxString GetIconFromMimeType(const wxString& mime);
```

Comment:
It fails to work; to fix it I would have to return to code that was reject the first time by kosenko.

@kosenko What do you think about this change request?

Tim S.